lib/book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+
+  attr_reader :title, :author, :publication_year
+
+  def initialize(book_info)
+    @book_info = book_info
```

Comment:
don't really need to track an instance variable for the hash of data; we don't ask that you reuse this later